File: src/main/java/com/example/serving_web_content/entity/Post.java
package com.example.serving_web_content.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "post")

public class Post {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "text", nullable = false)
    public String text;
}

File: models/post.py
class Post:
    def __init__(self, id, text, date_created):
        self.id = id
        self.text = text
        self.date_created = date_created

    def __str__(self):
        return "(id = %s, text = %s, date_created = %s)" % (
            self.id,
            self.text,
            self.date_created,
        )
